docs/docs/Components/components-data.md

```diff
@@ -262,28 +262,37 @@ Peruvian writer and Nobel Prize in Literature laureate Mario Vargas Llosa (pictu
 
 This component defines a webhook trigger that runs a flow when it receives an HTTP POST request.
 
+<!--TODO: Does this require auth in 1.5? -->
```

Comment:
Yes the code is updated already in the auth branch

Reply:
https://github.com/langflow-ai/langflow/pull/8721/files#diff-ff40327db3f0fd16743be7c714e149025127b3b9eb43ad72199992df6d1bd648R34